tests/dualtor/conftest.py

```diff
@@ -1,2 +1,23 @@
-from tests.common.dualtor.mux_simulator_control import mux_server_url, toggle_all_simulator_ports_to_rand_selected_tor # lgtm[py/unused-import]
+import pytest
+import logging
+import time
+from tests.common.dualtor.mux_simulator_control import mux_server_url, toggle_all_simulator_ports_to_upper_tor # lgtm[py/unused-import]
+from tests.common.dualtor.dual_tor_utils import lower_tor_host, upper_tor_host # lgtm[py/unused-import]
+
+CRM_POLL_INTERVAL = 1
+CRM_DEFAULT_POLL_INTERVAL = 300
+
+@pytest.fixture(scope='module', autouse=True)
+def set_crm_polling_interval(lower_tor_host):
```

Comment:
This doesn't look like the right place to put this fixture if it is only needed for crm test

Reply:
Thanks. I change the fixture to function level, and not autouse.